config/dev.webpack.config.js

```diff
@@ -4,15 +4,14 @@ const { GitRevisionPlugin } = require('git-revision-webpack-plugin');
 const config = require('@redhat-cloud-services/frontend-components-config');
 
 const webpackProxy = {
-    deployment: process.env.BETA ? 'beta/apps' : 'apps',
     useProxy: true,
     proxyVerbose: true,
-    debug: true,
     env: `${process.env.STAGE ? 'stage' : 'prod'}-${process.env.BETA ? 'beta' : 'stable'}`,
-    appUrl: process.env.BETA ? '/beta/insights/image-builder' : '/insights/image-builder',
 };
 
 const { config: webpackConfig, plugins } = config({
+    deployment: process.env.BETA ? 'beta/apps' : 'apps',
```

Comment:
Should the prod config also default to apps? Currently it is `beta/apps` or not default. Is default apps?

Reply:
Yea when we deploy to prod-stable it should also default to `apps`. I'll add this, one less thing to think about when we get there.